memphis/memphis.py

```diff
@@ -262,6 +292,7 @@ async def producer(self, station_name, producer_name, generate_random_suffix=Fal
                 raise MemphisError(create_res['error'])
 
             station_name_internal = get_internal_name(station_name)
+            self.station_schemaverse_to_dls[station_name_internal] = create_res['schemaverse_to_dls']
```

Comment:
You have to update the second parameter as well - the notification parameter, and send notifications only if needed